fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchReferenceTest.java

```diff
@@ -326,4 +341,58 @@ public void testSearchWithRelativePatientIdUsingIdMultipleTypes() {
                 "Multiple resource type matches found for logical ID '3002' for search parameter 'general-practitioner'.");
     }
 
+    @Test(groups = { "server-search-reference" }, dependsOnMethods = { "testCreatePatient" })
+    public void testSearchLogicalReferenceUsingId() {
+        // should find a match here because the search parameter only searches identifier of reference
```

Comment:
did you mean "shouldn't" ?
```suggestion
        // should NOT find a match here because the search parameter only searches identifier of reference
```

Reply:
Fixed.